src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/dossier/DossierContextFactory.kt

```diff
@@ -43,6 +43,10 @@ data class DossierContext(
       currentPrisonName,
       recall.bookingNumber!!
     )
+
+  fun includeWelsh(): Boolean {
```

Comment:
not sure if that's the best name, include suggests there could be others? 'welshOnly'?

Reply:
ah, I see now. `lduIsInWales`?

Reply:
I dunno what would really be better.  Everything else is already included.  This is the line of usage:
`createTableOfContentsDocumentMap(reasonsForRecallPdfBytes, recallId, dossierContext.includeWelsh())`
and the result is to add the Welsh doc.  🤷‍♂️